client/src/components/payment/payment-complete-modal.vue

```diff
@@ -150,3 +149,7 @@ export default class PaymentCompleteModal extends Mixins(NameRequestMixin, Payme
   }
 }
 </script>
+
+<style lang="scss" scoped>
+@import "@/assets/scss/base";
+</style>
```

Comment:
This style block is not used and could be deleted (optional).

Reply:
I have used class `button-blue`, which is defined in base.scss thats why I imported it

Reply:
I still think it doen't need to be here -- global styles are automatically picked up as they are not scoped. But I could be wrong.

Reply:
will test it